### PR TITLE
Fix migrations and remove icon composer package

### PR DESCRIPTION
**Pull request by Poseidon281**

### CODE REVIEW

File: src/Pages/Webhooks.php [hunk 1]
```diff
@@ -237,7 +240,7 @@ protected function isTablePaginationEnabled(): bool
 
     protected function getTableEmptyStateIcon(): ?string
     {
-        return 'gmdi-webhook';
+        return config('filament-webhook-server.navigation.icon');
```
Comment (MarJose123) [inline src/Pages/Webhooks.php]: Can you add some validation if the `navigation.icon` is existing in the config, and if not return the icon name instead.

**Review by MarJose123 — CHANGES_REQUESTED** [review]
All change is good, however I have one change request. thanks

**Review by Poseidon281 — COMMENTED** [review]
I reapplyed this commit because the TextInput causes a LiveWire issue.

File: src/Pages/Webhooks.php [hunk 2]
```diff
@@ -97,7 +99,7 @@ protected function getFormSchema(): array
         return [Grid::make(1)->schema(
             [
                 TextInput::make('name')->minLength(2)->maxLength(255)->required(),
-                Textarea::make('description')->required(),
+                TextInput::make('description'),
```
Comment (MarJose123) [inline src/Pages/Webhooks.php]: Can you revert this please. thanks.

Comment (Poseidon281) [reply src/Pages/Webhooks.php]: Done

**Pull request merged**